.github/workflows/ci.yml

```diff
@@ -239,7 +239,6 @@ jobs:
         linux-only:
           - ${{ github.event_name == 'pull_request' && !contains(github.head_ref, 'release-please') && !contains(github.event.pull_request.labels.*.name, 'full-ci') }}
         suite:
-          - firefox-cdp
```

Comment:
While the CDP code is still around and users can potentially still use it, should it be removed for all platforms or shall linux remain until all the CDP code for Firefox gets removed?

Reply:
I will remove it right after this.